.docker/paste/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM rustlang/rust:nightly
+
+MAINTAINER Jos Ahrens <zarthus@liefland.net>
+WORKDIR /app
+
+RUN apt update -y
+RUN apt install -y libsodium-dev cmake build-essential
+
+ADD bashrc.sh /root/.bashrc
+# TODO: get it from .env instead? but then it's not accessable by the container.
+ADD postgres_env /root/postgres_env
+RUN sed -E 's/[^=]+=//g' /root/postgres_env | tr "\n" " " | awk '{ print "db:5432:" $1 ":" $2 ":" $3 }' > /root/.pgpass && chmod 0600 /root/.pgpass
+RUN shred /root/postgres_env
+
+RUN cargo install diesel_cli --no-default-features --features postgres
+RUN cargo build
```

Comment:
should specify `--release`, I don't think it's a release build by default

Reply:
i was considering adding some sort of environment variable, as i'm currently writing the box to assist developers, not to run the software in production context.